.github/workflows/database-backup.yml

```diff
@@ -115,3 +115,68 @@ jobs:
           name: backup_sanitised
           path: backup_sanitised.sql
           retention-days: 3
+
+    job2:
```

Comment:
Can we give more explicit job names? Like backup_production and restore_to_preprod